sklearn/preprocessing/label.py

```diff
@@ -413,6 +416,10 @@ def label_binarize(y, classes, neg_label=0, pos_label=1, sparse_output=False):
     sparse_output : boolean (default: False),
         Set to true if output binary array is desired in CSR sparse format
 
+    force_matrix : boolean (default: False),
+        When true, binary targets are transformed to a 2d array;
```

Comment:
But computation `precision_recall_curve` require as many columns as you have classes, so even for binary proba prediction you get 2 but this gives you true labels as 1. 
The difficulty is described here #6191